flink-table/flink-table-runtime-blink/src/test/java/org/apache/flink/table/runtime/over/SumAggsHandleFunction.java

```diff
@@ -36,7 +36,7 @@ public SumAggsHandleFunction(int inputIndex) {
 	}
 
 	@Override
-	public void open(ExecutionContext ctx) throws Exception {
+	public void open(StateDataViewStore store) throws Exception {
```

Comment:
How about we expose this `StateDataViewStore` through `ExecutionContext`? In case of we might need some other information in the future

Reply:
Yes, it makes sense. But currently, `ExecutionContext` exposes `setCurrentKey`. `KeyedProcessFunction` doesn't have the ability to set current key, which means we can't create an instance of `ExecutionContext` in `GroupAggFunction`.  

So I'm planning to do this refactoring in [FLINK-12215](https://issues.apache.org/jira/browse/FLINK-12215) when we introducing a SQL own `ProcessFunction`.